### ISSUE TITLE
[shelly] Handler ShellyRelayHandler of thing...is linked although the handler was already disposed.

### ISSUE THREAD

**Issue by markus7017** [openhab/openhab-addons#8254]
Sometimes when unloading the binding the following messages appears
```
Handler ShellyRelayHandler of thing shelly:shelly25-relay:686ec1 tried checking if channel device#heartBeat is linked although the handler was already disposed.
```

**Comment by markus7017** [comment]
I added additional checks and didn't saw that again

**Comment by Hilbrand** [comment]
Can this be closed?

**Comment by markus7017** [comment]
fixed with #8198
